### DISCUSSION TITLE
Log4J

### DISCUSSION THREAD

**Discussion by mobile-solutions-ltd** [airsdk/Adobe-Runtime-Support#1479] [General]
Hi,

I've been asked by a customer for a statement about our App's status with respect to the Apache Log4J vulnerability.

Can you confirm please that it won't be an issue for AIR applications, or if it is please tell me which version of AIR we should use to republish our application not to have the vulnerability.

**Comment by ajwfrost** [comment]
AIR itself doesn't suffer from any exposure to this vulnerability. We don't use Log4j in any deliverables.
thanks

**Reply by mobile-solutions-ltd** [reply]
Great thank you
